test/test_cohort_defaults.py

```diff
@@ -32,9 +32,9 @@
 
 FILE_FORMAT_1 = "patient_format1_%s.vcf"
 
-def test_default_filter_fn():
+def test_default_filter_fn(summary_fn=variant_count):
```

Comment:
I don't know - i believe you had asked me to make this a function. I will force a failure to test.